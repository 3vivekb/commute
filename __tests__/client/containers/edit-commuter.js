/* global describe, expect, it */

import {mount} from 'enzyme'
import {mountToJson} from 'enzyme-to-json'
import React from 'react'
import {Provider} from 'react-redux'

import {makeMockStore, mockStores} from '../../test-utils/mock-data.js'

import EditCommuter from '../../../client/containers/edit-commuter'

const mockStore = makeMockStore(mockStores.init)

describe('Container > EditCommuter', () => {
  it('Create/Edit Commuter View loads (create or edit mode)', () => {
    // mount component
    const tree = mount(
      <Provider store={mockStore}>
        <EditCommuter />
      </Provider>
    )
    expect(mountToJson(tree.find(EditCommuter))).toMatchSnapshot()
  })

  it('Create/Edit Commuter View loads in edit mode', () => {
    throw new Error('unimplemented')
  })

  it('Create/Edit commuter', () => {
    throw new Error('unimplemented')
  })

  it('Delete commuter', () => {
    throw new Error('unimplemented')
  })

  it('Navigate back to Commuter Group View', () => {
    throw new Error('unimplemented')
  })
})
