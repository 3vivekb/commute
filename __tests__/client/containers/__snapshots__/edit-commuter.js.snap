exports[`Container > EditCommuter Create/Edit Commuter View loads (create or edit mode) 1`] = `
<Connect(EditCommuter)>
  <EditCommuter
    dispatch={[Function]}>
    <h1>
      Edit Site
    </h1>
  </EditCommuter>
</Connect(EditCommuter)>
`;
